tools/mo/openvino/tools/mo/back/pass_separator.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (C) 2018-2021 Intel Corporation
```

Comment:
marked as added

Reply:
It's a bug of github display. Indeed it's moving, I did it via `git mv`.

After merge git blame and history will be kept with no issues (i double checked that).